application/src/main/java/org/opentripplanner/model/plan/paging/cursor/PageCursorFactory.java

```diff
@@ -22,28 +22,47 @@ public class PageCursorFactory {
   private boolean wholeSearchWindowUsed = true;
   private ItinerarySortKey itineraryPageCut = null;
   private PageCursorInput pageCursorInput = null;
+  private Instant firstSearchLatestItineraryDeparture = null;
 
   private PageCursor nextCursor = null;
   private PageCursor prevCursor = null;
 
+  /**
+   * The search-window start and end is [inclusive, exclusive], so to calculate the start of the
+   * search-window from the last time included in the search window we need to include one extra
+   * minute at the end.
+   * <p>
+   * The value is set to 60 seconds because raptor operates in one minute increments.
+   */
+  private static final Duration SEARCH_WINDOW_END_EXCLUSIVITY_TIME_ADDITION = Duration.ofSeconds(
```

Comment:
Static final variables should be at the top of the class

Reply:
I moved the variable to the top and changed the wording from seconds -> minute